src/Tribe/Main.php

```diff
@@ -668,6 +668,17 @@ public function add_template_updates_check( $plugins ) {
 		return $plugins;
 	}
 
+	/**
+	 * Load the Event Tickets text domain after Tribe Common's.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function load_text_domain() {
+		return Tribe__Main::instance( $this )->load_text_domain( 'event-tickets', $this->plugin_dir . 'lang/' );
+	}
```

Comment:
@sc0ttkclark do you want to fire an action hook here, like Common does for us to link into, such as for ETP to load only after ET's?

Reply:
I really feel like we should hook all things directly to common hook, to avoid confusion.

Reply:
Yeah we don't need to hook in a specific order, common is the main one here that we want to run first (to standardize)